src/github/pull-request.ts

```diff
@@ -111,3 +95,29 @@ export default async (
 		jiraResponse?.status
 	);
 };
+
+const updateGithubIssues = async (github: GitHubClient | GitHubAPI, context: CustomContext, util, pullRequest) => {
+	const linkifiedBody = await util.unfurl(pullRequest.body);
+	if (!linkifiedBody) {
+		return;
+	}
+
+	context.log("Updating pull request");
+	const editedPullRequest = context.issue({
+		body: linkifiedBody,
+		id: pullRequest.id
```

Comment:
ID doesn't seem to be required, but it is in the tests so didn't pull it out incase there was some other reason its required that I didn't spot.

Reply:
yeah, pretty sure it's not.  Also, I'd remove the `context.issue` data object generator and build it yourself (you can look at my issue PR for an example).